src/Language.ts

```diff
@@ -146,11 +145,9 @@ export enum IssueKind {
  */
 export class Issue {
     constructor(private _span: Span, private _message: string, public kind: IssueKind) {
-        assert(_span !== null, "_span must not be null.");
-        assert(_span !== undefined, "_span must not be undefined.");
+        assert(_span, "_span must not be null/undefined");
```

Comment:
>_span [](start = 15, length = 5)

Why do we not need !IsNullOrUndefined here?

Reply:
OK, you changed it later, nevermind.

---
In reply to: [366153737](https://github.com/microsoft/vscode-azurearmtools/pull/443#discussion_r366153737) [](ancestors = 366153737)